protected/humhub/modules/user/models/Invite.php

```diff
@@ -110,7 +118,7 @@ public function selfInvite()
             $existingInvite->delete();
         }
 
-        if ($this->allowSelfInvite() && $this->validate() && $this->save()) {
+        if ($this->allowSelfInvite() && $this->validate() && $this->save(false)) {
```

Comment:
`$this->validate() && $this->save(false)` == `$this->save()`

Reply:
I think this won't work because when calling $this->validate() captcha will be automatically re-generated, and at the processing stage of $this->save() validation will be started again and won't be passed successfully. Maybe I am missing something, please advise.
http://joxi.ru/82QvdMJTjE0Q4A

Reply:
`$this->allowSelfInvite() && $this->save()` should do trick. save includes validate.

Reply:
Thank you. I fixed it.